modules/retail/pages/retail-uyuni-server-setup.adoc

```diff
@@ -0,0 +1,233 @@
+
+. Enter an email address to receive status notifications and click btn:[Next] to continue.
+{productname} can sometimes send a large volume of notification emails.
+You can disable email notifications in the {webui} after setup, if you need to.
+
+. Enter your certificate information and a password.
+Passwords must be at at least seven characters in length, and must not contain spaces, single or double quotation marks (``'`` or ``"``), exclamation marks (``!``), or dollar signs (``$``).
+Always store your passwords in a secure location.
++
+
+[IMPORTANT]
+====
+You must have the certificate password to set up a {productname} Proxy Server.
+====
+
+. Click btn:[Next] to continue.
+
+. From the menu:Uyuni Setup[Database Settings] screen, enter a database user and password and click btn:[Next] to continue.
+Passwords must be at at least seven characters in length, and must not contain spaces, single or double quotation marks (``'`` or ``"``), exclamation marks (``!``), or dollar signs (``$``).
+Always store your passwords in a secure location.
+
+. Click btn:[Next] to continue.
+
+. Click btn:[Yes] to run setup when prompted.
+
+. When setup is complete, click btn:[Next] to continue.
+You will see the address of the {productname} {webui}.
+
+. Click btn:[Finish] to complete {productname} setup.
+
+
+
+== Create the Main Administration Account
+
+This section covers how to create your organization's main administration account for {productname}.
+
+[WARNING]
+====
+The main administration account has the highest authority within {productname}.
+Ensure you keep access information for this account secure.
+
+We recommend that you create lower level administration accounts for organizations and groups.
+Do not share the main administration access details.
+====
+
+
+.Procedure: Setting Up the Main Administration Account
+
+. In your web browser, enter the address for the {productname} {webui}.
+This address was provided after you completed setup.
+For more information, see xref:retail:retail-uyuni-server-setup.adoc#retail-server-setup-yast[].
+
+. Sign in to the {webui}, navigate to the menu:Create Organization[Organization Name] field, and enter your organization name.
+
+. In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
+
+. Fill in the Account Information fields including an email for system notifications.
+
+. Click btn:[Create Organization] to finish creating your administration account.
+
+When you have completed the {productname} {webui} setup, you are taken to the menu:Home[Overview] page.
+
+
+
+== Synchronize the openSUSE Leap Channel
+
+Use [command]``spacewalk-common-channels`` to obtain all needed channels:
+
+.Procedure:
+. First the openSUSE Leap that is the parrent channel for all the following channels needed:
++
+----
+spacewalk-common-channels opensuse_leap15_2
+----
```

Comment:
This can probably go before the procedure, using a line something like "Before you begin, ensure you have access to the ``opensuse_leap15_2`` parent channel. Then the first step becomes "On the {productname} Server, at the command prompt, as root, register the parent channel" with the spacewalk-common-channels command.

Reply:
ok